src/main/java/com/google/alpollo/model/Song.java

```diff
@@ -33,4 +44,9 @@ public String getName() {
   public Long id() {
     return BIG_PRIME_NUMBER * artist.hashCode() + name.hashCode();
   }
+
+  /** For objectify and saving maps. */
```

Comment:
can you provide more info, please? why do we need this method and how objectify use it